packages/lesswrong/components/posts/PostsAnnualReviewMarketTag.tsx

```diff
@@ -0,0 +1,83 @@
+import { AnnualReviewMarketInfo } from '../../lib/annualReviewMarkets';
+import { useSingle } from '../../lib/crud/withSingle';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import React from 'react';
+import { useHover } from '../common/withHover';
+
+const styles = (theme: ThemeType) => ({
+  expectedWinner: {
+    color: theme.palette.review.winner,
+    border: '1px solid',
+    borderColor: theme.palette.review.winner,
+    borderRadius: '4px',
+    fontFamily: theme.typography.fontFamily,
+    width: 'fit-content',
+    display: 'inline-block',
+    padding: '5px',
+    // marginTop: 8,
+    // marginBottom: 8,
+    boxSizing: 'border-box',
+  },
+  preview: {
+    maxWidth: 400,
+  },
+});
+
+const PostsAnnualReviewMarketTag = ({ post, annualReviewMarketInfo, classes }: {
+  post: PostsWithNavigation | PostsWithNavigationAndRevision | PostsList,
+  annualReviewMarketInfo: AnnualReviewMarketInfo | null,
```

Comment:
generally prefer optional properties to be actually optional rather than unions with `null`